extensions/amp-access/amp-access.md

```diff
@@ -482,26 +482,6 @@ This section will cover a detailed explanation of the design underlying the amp-
 [12]: #metering
 [13]: #first-click-free
 
-## Validation errors
+## Validation
 
-The following lists validation errors specific to the `amp-access` tag
-(see also `amp-access` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/validator-amp-access.protoascii)):
```

Comment:
It's already split in the previous version as it is linked directly to the split section and not the full AMP validation specification. So the previous phrasing is already inaccurate for the link as it is not the entirety of the AMP validation specification. Would you prefer something like:
See [`amp-access` in it's AMP validator specification](....).

I just can't see having [AMP validator specification](...) when each link is different for the same blob of text that's being linked. IMHO, the link text should be different since the link is different.